src/utils.js

```diff
@@ -1143,3 +1143,12 @@ export function convertTypes(types, params) {
   });
   return params;
 }
+
+export function evaluateTimeout(auctionTimeout, adapterTimeout, adapterName = 's2sConfig') {
+  if (auctionTimeout <= adapterTimeout) {
```

Comment:
In general I'm ok with shaving the timeout, but unclear 75% is the right number for every case. How about `s2sConfig.timeoutBufferPercent`?

Shouldn't this be `auctionTimeout * PERCENT <= adapterTimeout`?

Consider these scenarios:

auctionTimeout | original adapterTimeout | new adapterTimeout | Notes
-----|-------|--------|-----
1000 | 600 | 600 | original value under PERCENT
1000 | 800 | 750 | should pull adapterTimeout a little lower. Current code wouldn't notice this.
1100 | 1000 | 825 | current alg. also wouldn't notice this scenario

Reply:
thanks. Jason is out this week so will wait for him to get back and update this.

Reply:
@bretg 

I was thinking we could one day extend the use of the `evaluteTimeout` function to also work with the client-side adapters as well.  So instead of setting the coefficient value in the `s2sConfig`, what if we created another variable in the general `setConfig` to house the coefficient (ie similar to `bidderTimeout`)?  That way it would generally available and consistent throughout the auction.

As a follow-up either way - if we were to set the value through a config, should we have a default/backup value setup and if so, what value should it be if not 75%?  I set it to 75% as I thought it was a reasonable chunk of the auction's time to allow adapters to respond in most cases.

I see your point about adjusting the if logic, I'll make those changes.

Reply:
@bretg Any thoughts on the above?